app/controllers/tweets_controller.rb

```diff
@@ -1,4 +1,73 @@
+require 'pry'
 class TweetsController < ApplicationController
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect '/login'
+    end
+  end
 
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
 
+  post '/tweets/new' do
```

Comment:
This action should be `post '/tweets`. `get '/tweets/new'` does not typically have a corresponding `post '../new'` route.